doc/prop.xml

```diff
@@ -8,6 +8,106 @@
 #############################################################################
 ##
 
+<#GAPDoc Label="IsPartialOrderDigraph">
+<ManSection>
+  <Prop Name="IsPartialOrderDigraph" Arg="digraph"/>
+  <Returns><K>true</K> or <K>false</K>.</Returns>
+  <Description>
+    This is a synonym for <Ref Prop="IsReflexiveDigraph"/> and
+    <Ref Prop="IsAntisymmetricDigraph"/> and <Ref Prop="IsTransitiveDigraph"/>.
+
```

Comment:
On reflection, you've implicitly defined the partial order to be `x >= y` if and only if `x -> y` is an edge. So, for instance, the digraph `Digraph([[1, 2, 3], [2], [3]])`, which looks like (ignoring the loops):

```
   --> 2
 /
1 
 \
   --> 3
```
is, according to your code below, a join semilattice, but not a meet semilattice. For what it's worth, I would expect it to be the other way round, although I accept that it's arbitrary. However there's probably a well-established convention in use.